.github/workflows/test.sh

```diff
@@ -22,12 +22,7 @@ conda activate test-env
 if [ "$USE_CONDA" = "Yes" ]; then
 
     if [ "${1}" = "pyqt5" ]; then
-        if [ "${PYQT5_VERSION:0:4}" = "5.15" ]; then
-            # Must specify versions individually with PyQt 5.15 to avoid compat issues with Python 3.11
-            conda install -q qt-main=${PYQT5_QT_VERSION} qt-webengine=${PYQT5_QT_VERSION} pyqt=${PYQT5_VERSION}
-        else
-            conda install -q qt=${PYQT5_QT_VERSION} pyqt=${PYQT5_VERSION}
-        fi
+        conda install -q qt=${PYQT5_QT_VERSION} pyqt=${PYQT5_VERSION}
     elif [ "${1}" = "pyside2" ]; then
         conda install -q qt=${PYSIDE2_QT_VERSION} pyside2=${PYSIDE2_VERSION}
```

Comment:
```suggestion
        conda install -q qt-main=${PYSIDE2_QT_VERSION} qt-webengine=5.15 pyside2=${PYSIDE2_VERSION}
```

Reply:
See above—because `qt-main` is only available for Qt 5.15.x, so we'd have to add the same fragile, complex, duplicative and tedious to maintain branching logic that we had above to here as well if we were to do this.

Reply:
To note, that is indeed exactly what we do for PySide6, and will presumably be the case for PyQt6, as being a whole separate matrix job, package, and variables, the logic branch is much more justifiable and required anyway.